processor/collector.py

```diff
@@ -85,6 +85,14 @@ def _analyzer(self) -> None:
             self.get_data()
             if self.analyze_as_needed():
                 self.rotary.alarms = self.alarms
+                if "Current Setting" in self.rotary:
+                    setting = self.rotary["Current Setting"]
+                    RR = self.cumulative.get("RR", 0.0)
+
+                    setting.from_processor(
+                        F=[1.5] * 9, P=[3.5] * 9, RR=[RR] * 9,
```

Comment:
@jpivarski And here's where the values get set. Currently I fill F with 1.5, P with 3.5, and RR with the 1 (3?) breath RR.